src/Chip/Chip.js

```diff
@@ -0,0 +1,211 @@
+// @flow weak
+
+import React, { cloneElement, isValidElement, PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import keycode from 'keycode';
+import DeleteIcon from '../svg-icons/cancel';
+import { emphasize, fade } from '../styles/colorManipulator';
+
+export const styleSheet = createStyleSheet('Chip', (theme) => {
+  const { palette, shadows, transitions } = theme;
+  const height = 32;
+  const backgroundColor = emphasize(palette.background.default, 0.12);
+  const deleteIconColor = fade(palette.text.primary, 0.26);
+  return {
+    root: {
+      fontFamily: 'inherit', // Override `button` default system font
+      fontSize: 13,
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height,
+      color: palette.getContrastText(backgroundColor),
+      backgroundColor,
+      borderRadius: height / 2,
+      whiteSpace: 'nowrap',
+      width: 'fit-content',
+      transition: transitions.create(),
+      cursor: 'default', // label will inherit this from root, then `clickable` class overrides this for both
+      outline: 'none', // No outline on focused element in Chrome (as triggered by tabIndex prop)
+      border: 'none', // Remove `button` border
+      padding: 0, // Remove `button` padding
+    },
+    clickable: {
+      '&:hover, &:focus': {
+        backgroundColor: emphasize(backgroundColor, 0.08),
+      },
+      '&:active': {
+        boxShadow: shadows[1],
+        backgroundColor: emphasize(backgroundColor, 0.12),
+      },
+      cursor: 'pointer',
+    },
+    deletable: {
+      '&:focus': {
+        backgroundColor: emphasize(backgroundColor, 0.08),
+      },
+    },
+    avatar: {
+      marginRight: -4,
+      width: 32,
+      height: 32,
+      fontSize: 16,
+    },
+    avatarChildren: {
+      width: 19,
+      height: 19,
+    },
+    label: {
+      paddingLeft: 12,
+      paddingRight: 12,
+      userSelect: 'none',
+      whiteSpace: 'nowrap',
+      cursor: 'inherit',
+    },
+    deleteIcon: {
+      color: deleteIconColor,
+      '&:hover': {
+        color: fade(deleteIconColor, 0.4),
+      },
+      cursor: 'pointer',
+      margin: '0 4px 0 -8px',
+    },
+  };
+});
+
+/**
+ * Chips represent complex entities in small blocks, such as a contact.
+ *
+ * ```jsx
+ * <Chip avatar={<Avatar>} label="Label text" />
+ * ```
+ */
+export default function Chip(props, context) {
+  const {
+    avatar: avatarProp,
+    className: classNameProp,
+    deleteIconClassName: deleteIconClassNameProp,
+    label,
+    labelClassName: labelClassNameProp,
+    onClick,
+    onKeyDown,
+    onRequestDelete,
+    ...other
+  } = props;
+
+  let chipRef;
+
+  const handleDeleteIconClick = (event) => {
```

Comment:
Right, my bad, that's not going to change anything.
Using a class on the other hand could help.